online-shop/src/components/Products.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import '../css/Products.css';
+import Product from './Product';
+import { withStyles, createStyles, Theme, ThemeProvider } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import { createMuiTheme } from '@material-ui/core/styles';
+
+const myCustomTheme = createMuiTheme({
+  palette: {
+    primary: {
+      main: '#a01441',
+    },
+  },
+});
+
+const MyCellType = withStyles((theme: Theme) =>
+  createStyles({
+    head: {
+      backgroundColor: theme.palette.primary.main,
+      color: theme.palette.common.white,
+      fontSize: 25,
+      fontStyle: 'bold',
+    }
+  }),
+)(TableCell);
+
+function Products(props: { products: any }) {
+  const products = props.products;
```

Comment:
In loc de asta, foloseste destructuring pe obiectul de props, in linia 31.
https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0